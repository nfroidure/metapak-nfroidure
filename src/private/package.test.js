'use strict';

const packageTransformer = require('./package');

describe('private', () => {
  describe('Package transformer', () => {
    it('should work', () => {
      expect(packageTransformer({})).toMatchSnapshot();
    });
  });
});
