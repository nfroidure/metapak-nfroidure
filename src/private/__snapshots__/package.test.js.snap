// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`private Package transformer should work 1`] = `
{
  "private": true,
}
`;
